src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Email } from '../Email/Email';
+import { Section } from '../Section/Section';
+import { Column } from '../Column/Column';
+import { Button } from './Button';
+
+export default {
+  component: Button,
+} as ComponentMeta<typeof Button>;
+
+//“template” of how args map to rendering
+const Template: ComponentStory<typeof Button> = (args) => (
+  <Email>
+    <Section>
+      <Column>
+        <Button {...args} />
+      </Column>
+    </Section>
+  </Email>
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
```

Comment:
Add stories for secondary button also

Reply:
Added stories for `Primary` and `Secondary` buttons